src/main/java/com/iridium/iridiumskyblock/configs/Messages.java

```diff
@@ -18,5 +18,7 @@
     public String islandSchematicNotFound = "%prefix% &7No schematic with that name exists.";
     public String islandDeleted = "%prefix% &7Your island has been deleted.";
     public String teleportingHome = "%prefix% &7Teleporting Home...";
+    public String setHome = "%prefix% &7You have set your home to this location.";
```

Comment:
Change to Your Island  home has been set

Reply:
oh, i was copied it from old messages -,-